components/dialog/stories/dialog.stories.js

```diff
@@ -66,17 +120,154 @@ export default {
 	},
 };
 
+const ExampleContent = Typography({
+	semantics: "body",
+	size: "m",
+	content: [
+		"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Auctor augue mauris augue neque gravida. Libero volutpat sed ornare arcu. Quisque egestas diam in arcu cursus euismod quis viverra. Posuere ac ut consequat semper viverra nam libero justo laoreet. Enim ut tellus elementum sagittis vitae et leo duis ut. Neque laoreet suspendisse interdum consectetur libero id faucibus nisl. Diam volutpat commodo sed egestas egestas. Dolor magna eget est lorem ipsum dolor. Vitae suscipit tellus mauris a diam maecenas sed. Turpis in eu mi bibendum neque egestas congue. Rhoncus est pellentesque elit ullamcorper dignissim cras lobortis."
+	]
+});
+
+/**
+ * The default size for dialog is medium.
+ */
 export const Default = Template.bind({});
 Default.args = {
 	heading: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
 	showModal: true,
 	content: [
-		() => Typography({
-			semantics: "body",
-			size: "m",
-			content: [
-				"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Auctor augue mauris augue neque gravida. Libero volutpat sed ornare arcu. Quisque egestas diam in arcu cursus euismod quis viverra. Posuere ac ut consequat semper viverra nam libero justo laoreet. Enim ut tellus elementum sagittis vitae et leo duis ut. Neque laoreet suspendisse interdum consectetur libero id faucibus nisl. Diam volutpat commodo sed egestas egestas. Dolor magna eget est lorem ipsum dolor. Vitae suscipit tellus mauris a diam maecenas sed. Turpis in eu mi bibendum neque egestas congue. Rhoncus est pellentesque elit ullamcorper dignissim cras lobortis."
-			]
-		}),
+		() => ExampleContent
+	],
+};
+
+/*
+ * Stories for "docs only."
+ */
+export const DefaultSmall = Template.bind({});
+DefaultSmall.tags = ["docs-only"];
```

Comment:
```suggestion
DefaultSmall.tags = ["autodocs", "!dev"];
```

Can you update these tags with our new tagging knowledge when you get a chance?

Reply:
We should also exclude these from chromatic snapshots and create a combined snapshot on Default.

Reply:
Corrected tags can be found here: https://github.com/adobe/spectrum-css/pull/2833/commits/8305a2d4a376316b28340ecc6756684715bd4451
^ that should take care of the tagging in all 4 components

Excluded Chromatic snapshots: 
- dial: https://github.com/adobe/spectrum-css/pull/2833/commits/d547a48e60ae1537daaa42d6547719f91a5dfdfa
- dialog: https://github.com/adobe/spectrum-css/pull/2833/commits/4cf1b8aed2b20f8eb9463958c70f06d866627bb2

Reply:
> We should also exclude these from chromatic snapshots and create a combined snapshot on Default.

@castastrophe I created the chromatic group for dialog in the s2 branch ([PR](https://github.com/adobe/spectrum-css/pull/2860) or [preview](https://pr-2860--spectrum-css.netlify.app/preview/?path=/docs/components-dialog--docs&globals=testingPreview:!true)). We had decided that adding chromatic coverage in these docs migration branches was making the PRs quite large, so removed that criteria from [the ticket](https://jira.corp.adobe.com/browse/CSS-759). 

I can either take this feedback and apply it over on that S2 branch (with regards to making sure the chromatic group is in the template.js file), or do something like we did for menu, where we had a separate PR completely for the chromatic coverage (so that the chromatic baselines are in `main` sooner. Do you have a preference?